src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.ManagedIdentity.cs

```diff
@@ -80,17 +83,36 @@ internal async Task<IManagedIdentityApplication> GetOrBuildManagedIdentityApplic
         /// Creates a managed identity client application.
         /// </summary>
         /// <param name="managedIdentityId">Indicates if system-assigned or user-assigned managed identity is used.</param>
+        /// <param name="capabilities">Indicates the capabilities of the managed identity application.</param>
         /// <param name="enablePiiLogging">Indicates if logging that may contain personally identifiable information is enabled.</param>
         /// <returns>A managed identity application.</returns>
-        private IManagedIdentityApplication BuildManagedIdentityApplication(ManagedIdentityId managedIdentityId, bool enablePiiLogging)
+        private IManagedIdentityApplication BuildManagedIdentityApplication(
+            ManagedIdentityId managedIdentityId,
+            IEnumerable<string>? capabilities,
+            bool enablePiiLogging)
         {
-            return ManagedIdentityApplicationBuilder
+            ManagedIdentityApplicationBuilder miBuilder = ManagedIdentityApplicationBuilder
                 .Create(managedIdentityId)
                 .WithLogging(
                     Log,
                     ConvertMicrosoftExtensionsLogLevelToMsal(_logger),
-                    enablePiiLogging: enablePiiLogging)
-                .Build();
+                    enablePiiLogging: enablePiiLogging);
+
+            if (capabilities?.Any() == true)
```

Comment:
Capabilities do not pass in MSI flow. Need to expose it to ManagedIdentityOptions

Reply:
PR - https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/pull/179

Reply:
I don't know why this is needed, it's already surfaced in the other options: https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/blob/main/src/Microsoft.Identity.Abstractions/ApplicationOptions/MicrosoftEntraApplicationOptions.cs#L64

Reply:
Closed the PR, and added unit tests to use, 

```
// Enable capabilities cp1,cp2
factory.Services.Configure<MicrosoftIdentityApplicationOptions>(opts =>
    opts.ClientCapabilities = ["cp1", "cp2"]);
```